draft-kazuho-httpbis-priority.md

```diff
@@ -215,23 +215,25 @@ extensible priority scheme when and only when all the requests that are to be
 sent on that backend connection originates from one client-side connection that
 has negotiated the use of the extensible priority scheme (see {{fairness}}).
 
-# The Priority HTTP Header Field
+# Urgency Prioritization
```

Comment:
I'm starting to think that this name is confusing.

In my view, this section discusses two things: 1) the extensible scheme using the Structured Headers, 2) the urgency and the progressive parameters.

Even if we are to consider calling (2) as "Urgency Prioritization" to be okay, I think it's wired to refer to 1 using the same name.

Reply:
Agreed, including the structured header part in this section is a bit odd.

Reply:
Reconsidering this, I think we need to agree on what we expose as layers, based on how we think that the extensibility is going to be used. I've opened #86. PTAL.